universal-application-tool-0.0.1/app/services/applicant/ApplicantServiceImpl.java

```diff
@@ -397,6 +392,11 @@ private void stageEnumeratorUpdates(
       throw new PathNotInBlockException(block.getId(), unknownUpdates.iterator().next().path());
     }
 
+    // Before adding anything, if only metadata is being stored then delete it
```

Comment:
suggestion (blocking): add something like "We cannot put an array of entities at the question path if a JSON object with metadata already exists" for clarity, since this is a weird case that isn't very obvious

Reply:
fixed!